typescript-express-reviews/tests/index.test.ts

```diff
@@ -9,9 +9,9 @@ before(async function() {
 
   // Create namespace, otherwise get "Unknown namespace test, you must create it first"
   // @ts-ignore
-  await mongoose.connection.client.httpClient.post('/v2/schemas/namespaces', {
+  /*await mongoose.connection.client.httpClient.post('/v2/schemas/namespaces', {
```

Comment:
Feel free to delete this code. This line is no longer necessary after https://github.com/stargate/stargate-mongoose/pull/40/files#diff-f86914c187187f64bd501f514851e2330d2166aba32e09a9e93b2c8b215121d9

Reply:
Done